packages/stratimikos/adapters/belos/tpetra/README.md

```diff
@@ -0,0 +1,36 @@
+# Belos Tpetra Preconditioning in Stratimikos
+
+The adapters in this directory allow a (unpreconditioned) Belos solver using Tpetra linear algebra to be called from a Stratimikos input deck as a preconditioner to another Belos solver.  
```

Comment:
What about the preconditioned case?

Reply:
We are discussing this in person... need to meet with the rest of the multiprecision team.